libs/pika/async_mpi/tests/unit/mpi_async_storage.cpp

```diff
@@ -514,8 +487,8 @@ int main(int argc, char* argv[])
     nws_deb<6>.debug(3, "Calling pika::init");
     pika::init_params init_args;
     init_args.desc_cmdline = cmdline;
-    // Set the callback to init thread_pools
-    init_args.rp_callback = &init_resource_partitioner_handler;
+    // Tell init to create an mpi thread pool if needed
+    init_args.resource::polling_pool_creation_mode = ::pika::resource::polling_pool_pika_decides;
```

Comment:
Shouldn't this be removed?

Reply:
correct. I forgot to check the other tests properly it seems.